src/structures/Message.js

```diff
@@ -155,6 +148,13 @@ class Message {
      * @private
      */
     this._edits = [];
+
+    if (data.member && this.guild && this.author) {
+      if (!this.guild.members.has(data.member.id)) {
+        this.guild._addMember(this.author, false);
```

Comment:
You should pass the member info here, the method can not make use of a user.
`Object.assign(data.member, { user: this.author });`